### PR TITLE
Allow force install of node and version specification.

### PR DESCRIPTION
**Pull request by stackoverjoe**

### CODE REVIEW

**Review by splatteredbits — CHANGES_REQUESTED** [review]

File: Test/InstallNode.Tests.ps1 [hunk 1]
```diff
@@ -6,27 +6,100 @@ Set-StrictMode -Version 'Latest'
 
 [Whiskey.Context]$context = $null
 
+$script:latestNodeVersion = 
+    Invoke-RestMethod -Uri 'https://nodejs.org/dist/index.json' | 
+    Sort-Object | 
+    Select-Object -ExpandProperty SyncRoot | 
+    Where-Object{ $_.lts -ne $false } |
```
[the remark this answers is not in the document]
Comment (stackoverjoe) [reply Test/InstallNode.Tests.ps1:13]: Done.

File: Test/InstallNode.Tests.ps1 [hunk 2]
```diff
@@ -6,27 +6,100 @@ Set-StrictMode -Version 'Latest'
...
 
 function ThenNodeInstalled
 {
-    Test-Path -Path (Join-Path -Path $script:testRoot -ChildPath '.node') -PathType Container | Should -BeTrue
+    param(
+        [String]$Version
+    )
+
+    $nodePath = Join-Path -Path $script:testRoot -ChildPath '.node'
+    Test-Path -Path $nodePath -PathType Container | Should -BeTrue
```
[the remark this answers is not in the document]
Comment (stackoverjoe) [reply Test/InstallNode.Tests.ps1:31]: Done.

File: Test/InstallNode.Tests.ps1 [hunk 3]
```diff
@@ -6,27 +6,100 @@ Set-StrictMode -Version 'Latest'
...
+    if($Version)
+    {
+        Get-ChildItem -Path $nodePath | Where-Object { $_.Name -match '\d+\.\d+\.\d+' } | Select-Object -ExpandProperty Name
+        $Version -match $Matches[0] | Should -BeTrue
```
[the remark this answers is not in the document]
Comment (stackoverjoe) [reply Test/InstallNode.Tests.ps1:35]: Done.

File: Test/InstallNode.Tests.ps1 [hunk 4]
```diff
@@ -6,27 +6,100 @@ Set-StrictMode -Version 'Latest'
...
+    $nodePath = Join-Path -Path $script:testRoot -ChildPath '.node'
     $script:context = New-WhiskeyTestContext -ForDeveloper -ForBuildRoot $script:testRoot
+    $parameters = @{
+        'Version' = $Version
+    }
+    Invoke-WhiskeyTask -TaskContext $context -Parameter $parameters -Name 'InstallNode' 
+    $script:fileCreatedTime.add(((Get-ChildItem -Path $nodePath).CreationTime | Select-Object -ExpandProperty ticks))
+}
+
+function WhenInstallingNodeForce
```
[the remark this answers is not in the document]
Comment (stackoverjoe) [reply Test/InstallNode.Tests.ps1:54]: Done.

File: Test/InstallNode.Tests.ps1 [hunk 5]
```diff
@@ -6,27 +6,100 @@ Set-StrictMode -Version 'Latest'
...
+    $parameters = @{
+        'Force' = $true
+        'Version' = $Version
+    }
     Invoke-WhiskeyTask -TaskContext $context -Parameter $parameters -Name 'InstallNode'
 }
 
+function ThenNodeFolderDidNotChange
+{
+    @( $filecreatedTime | Select-Object -Unique ).Count -eq 1 | Should -BeTrue
```
[the remark this answers is not in the document]
Comment (stackoverjoe) [reply Test/InstallNode.Tests.ps1:68]: Done.

File: Whiskey/Tasks/InstallNode.ps1 [hunk 6]
```diff
@@ -2,7 +2,17 @@ function Install-Node
 {
     [Whiskey.Task('InstallNode')]
     [Whiskey.RequiresTool('Node', PathParameterName='NodePath')]
-    [CmdletBinding()]
     param(
+        [Parameter(Mandatory)]
```
[the remark this answers is not in the document]
Comment (stackoverjoe) [reply Whiskey/Tasks/InstallNode.ps1:6]: Done.

**Review by splatteredbits — APPROVED** [review]

**Pull request merged**